app/featuresmodel.cpp

```diff
@@ -57,28 +56,21 @@ QVariant FeaturesModel::data( const QModelIndex &index, int role ) const
         return QVariant( feat.feature().id() );
       return QVariant( title );
     }
-    case FeatureId:
-      return QVariant( feat.feature().id() );
-    case Description:
-      return QVariant( QString( "description" ) );
-    case GeometryType:
-      return QVariant( feat.feature().geometry().type() );
-    default:
-      return QVariant();
+    case FeatureId: return QVariant( feat.feature().id() );
+    case Description: return QVariant( QString( "description" ) );
```

Comment:
Let's simply use "Feature ID %1" .arg( feat.feature().id() ) ... it will be better than just "description" :-)

Reply:
I wanted to change this in #796, but sure I can change it here